src/game_constants.h

```diff
@@ -97,4 +97,10 @@ constexpr double accuracy_goodhit  = 0.5;
 constexpr double accuracy_standard = 0.8;
 constexpr double accuracy_grazing  = 1.0;
 
+/** Minimum item damage output of relevant type to allow using with relevant weapon skill */
+#define MELEE_STAT 5
```

Comment:
That name is really ambiguous.
The old option - having `is_cutting_weapon` etc. was better.
Could be combined into `is_weapon( damage_type )` - this would be both clear and centralized.